File: Programmers/로또의_최고_순위와_최저_순위.py
# 로또의_최고_순위와_최저_순위

def solution(lottos, win_nums):
    answer = []
    x = lottos.count(0)
    cnt = 0
    for i in win_nums:
        if i in lottos:
            cnt += 1
    ans = 6
    if cnt == 6:
        ans = 1
    elif cnt == 5:
        ans = 2
    elif cnt == 4:
        ans = 3
    elif cnt == 3:
        ans = 4
    elif cnt == 2:
        ans = 5

    if ans - x > 0:
        answer.append(ans-x)
    else:
        answer.append(1)
    answer.append(ans)
        
    return answer

File: Programmers/로또의_최고_순위와_최저_순위.js
function solution(lottos, win_nums) {
  let answer = [];
  let cnt = 0;
  const rank = [6,6,5,4,3,2,1];
  let x = lottos.filter(element => 0 == element).length;
  for (let step = 0; step < lottos.length; step++) {
      for (let i = 0; i < lottos.length; i++){
          if (lottos[i] == win_nums[step]) {
              cnt += 1;
          }
      }
  };
  answer.push(rank[cnt+x],rank[cnt]);

  return answer;
}